etcd/etcd_test.go

```diff
@@ -310,6 +311,50 @@ func TestReceiveInsert(t *testing.T) {
 	}
 }
 
+func TestReceiveExpire(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+	}
+	if err := etcdtest.DeleteAll(); err != nil {
+		t.Error(err)
+		return
+	}
+
+	c, err := New(Options{etcdtest.Urls, "/skippertest", 0, false, "", "", ""})
+	if err != nil {
+		t.Error(err)
+		return
+	}
+
+	c.LoadAll()
+
+	// Will expire after a TTL of 1 second
+	etcdtest.PutDataToTTL("/skippertest", "pdp", `Path("/pdp") -> "https://expire.example.org"`, 1)
+
+	// Wait until the route is expired with a timeout of 5 seconds
+	timeout := time.After(5 * time.Second)
+	for {
+		select {
+		case <-timeout:
+			t.Errorf("Timeout: route should expire after 1 second but did not expire within 5 seconds")
```

Comment:
minor: Maybe t.Fatalf() is more appropriate here

Reply:
I mean use t.Fatalf() whenever no error is expected and the check is not related to the change and use t.Errorf() to check errors related to the change.

Reply:
Thanks for the suggestion.
I now try to use t.Error in the context of the change and t.Fatal otherwise, e.g. while preparing the test.